app/controllers/report_controller.rb

```diff
@@ -274,6 +274,19 @@ def title
 
   private
 
+  # Get string with unavailable fields for displaying note about them while adding/editing report
+  def unavailable_fields_for_model
+    case @edit[:new][:model]
```

Comment:
Instead of testing for the `@edit[:new][:model]` I would make this method more *universal* by using an argument instead.